tests/utils/dockercli/containers.go

```diff
@@ -86,3 +86,24 @@ func GetVolumeProperties(volumeName, hostName string) (string, error) {
 	return ssh.InvokeCommand(hostName, cmd)
 }
 
+// WriteToContainer write data to a file in the running container
+func WriteToContainer(ip, containerName, volPath, fileName, data string) (string, error) {
```

Comment:
I see. It is better to call this out at the method documentation to avoid such confusion in the future. 

nit: `write data to a file in the running container` is not sufficient in my opinion ... something like `this util writes data from an existing running container  as per passed container name` in short some more elaborative.